tools/Unix.mk

```diff
@@ -618,6 +619,13 @@ bootloader:
 clean_bootloader:
 	$(Q) $(MAKE) -C $(ARCH_SRC) clean_bootloader
 
+# debug_info: Parse nxdiag example output file (sysinfo.h) and print
+
+debug_info:
```

Comment:
why not generate sysinfo.h inside apps/system/nxdiag directly?

Reply:
I think there is a missing point here. The point of this PR is about fetching system information without using any additional configuration or enabling something for diagnostic. If a user has an issue, they can run `make debug_info` command to get diagnostic information during issue reporting (at least out plan is that) on their configuration. For that reason I added target into there (I tried other solution but could not get any result without adding this target on build system).

Target uses sysinfo.h because it was implemented already, I just modified it to get system data.

Reply:
@xiaoxiang781216 the intention here is to add this command `make debug_info` into the Issue's template. 

Whenever an issue is reported, the idea is to gather data from the user's environment (something that we already do in a non-standardized manner) by suggesting they run this command in the nuttx (OS) folder and paste it to the issue template. 

Please note that the `nxdiag` can be used as an application built into the firmware, but also as a tool because `sysinfo.h` is generated before compilation. This PR intends to use it as a tool to get standardized info about the user's environment.

The `nxdiag` application isn't specific for Espressif and we are trying to make it part of the NuttX debugging ecosystem.

Reply:
If so, I think that the better approach is splitting the part of nxdiag script to nuttx/tools/, so the new debug_info can work without apps. On the other side, nxdiag can invoke the script inside nuttx/tools directly.

Reply:
So, we will move nxdiag script to the nuttx/tools and let nxdiag app run on nuttx/tools too.

Reply:
no, nxdiag still keep in apps/system, only the script moves to nuttx/tools.

Reply:
Sorry for misunderstanding. I mean `parse_sysinfo.py` and `host_sysinfo.py`

Reply:
Updated